doc/user/content/get-started.md

```diff
@@ -48,31 +48,27 @@ We'll start with some real-time data produced by Materialize's built-in [load ge
 
     ```sql
     CREATE SOURCE auction_house
-    FROM LOAD GENERATOR AUCTION
-    FOR ALL TABLES;
+        FROM LOAD GENERATOR AUCTION
+        FOR ALL TABLES
+        WITH (SIZE '3xsmall');
```

Comment:
I didn't know that the `=` is optional (e.g. `SIZE = '3xsmall'`). On a quick skim, we've been using the variant with `=` more frequently.